views/signup.ejs

```diff
@@ -13,30 +13,39 @@
         <% messages.errors.forEach( el => { %>
             <div class="alert alert-danger"><%= el.msg %></div>
         <% }) %>    
-    <% } %>
-
- <div class="flex-container"> 
-        <form action="/signup" method="POST" id="new-sign-up">
-            <h1 class="credentials-header">Credentials</h1>
-            <div class="flex-child">
-                <input type="text" name="userName" placeholder="Enter Your User Name">
-                <input type="email" name="email" placeholder="Enter Your Email">
-                <input type="password" name="password" placeholder="Create Your Password">
-                <input type="password" name="confirmPassword" placeholder="Confirm Your Password">
-                <a href="/" class="back-btn">Back</a>
+    <% } %> 
+        <div class="signup-container"> 
+            <div class="form-container"> 
+                <div class="flex-child-one">
+                    <h2 class="credentials-header">Credentials</h2>
+                    <form action="/signup" method="POST" id="new-sign-up">
+                        <div class="form-left">
+                            <input type="text" name="userName" placeholder="Enter Your User Name">
+                            <input type="email" name="email" placeholder="Enter Your Email">
+                            <input type="password" name="password" placeholder="Create Your Password">
+                            <input type="password" name="confirmPassword" placeholder="Confirm Your Password">
+                            <button href="/" class="back-btn">Back</button>
+                        </div>
+                    </form>   
+                </div>
+                <hr style=" height: 350px; margin-top: 35px;">
+                <div class="flex-child-two">
+                    <h2>About Me</h2> 
+                     <form action="/signup" method="POST" id="new-sign-up">
```

Comment:
Splitting the forms into two causes a breaking change, as only the data in the second form are submitted with the request.